yellowbrick/draw.py

```diff
@@ -92,7 +92,7 @@ def manual_legend(g, labels, colors, **legend_kwargs):
 
 
 def bar_stack(data, ax=None, labels=None, ticks=None, colors=None, 
-              orientation='vertical', colormap=None, **kwargs):
+              orientation='vertical', colormap=None, legend_kws=None, **kwargs):
```

Comment:
In terms of argument ordering, could we put `orientation='vertical'` after `colormap=None`? I think that groups the  args a bit nicer (also update the order in the docstring).

Reply:
Will do so!